aiohue/clip.py

```diff
@@ -0,0 +1,32 @@
+class Clip:
+    """Represent Hue clip."""
+
+    def __init__(self, request_2):
+        self._request_2 = request_2
+
+    async def next_events(self):
+        """Note, this method will be pending until next event."""
+        return await self._request_2("get", "eventstream/clip/v2")
+
+    async def resources(self):
+        """Fetch resources from Hue.
+
+        Available types:
+
+        homekit
+        device
+        bridge
+        zigbee_connectivity
+        entertainment
+        light
+        bridge_home
+        grouped_light
+        room
+        scene
+        """
+        return await self._request_2("get", "clip/v2/resource")
+
+    @staticmethod
+    async def initialize(request_2):
+        raw = await request_2("get", "clip/v2/resource")
+        return Clip(raw, request_2)
```

Comment:
```suggestion
        return cls(raw, request_2)
```

Reply:
This call doesn't match the signature of the class?

Reply:
Yeah, it's old. Removing it.